tests/tck/features/match/VariableLengthPattern.feature

```diff
@@ -310,7 +310,7 @@ Feature: Variable length Pattern match (m to n)
   Scenario: multi-steps and filter by node properties
     When executing query:
       """
-      MATCH (v:player{name: 'Tim Duncan'})-[e1:like*1..2]-(v2{name: 'Tony Parker'})-[e2:serve]-(v3{name: 'Spurs'})
+      MATCH (v:player{name: 'Tim Duncan'})-[e1:like*1..2]-(v2:player{name: 'Tony Parker'})-[e2:serve]-(v3:team{name: 'Spurs'})
```

Comment:
Please fix similar issue related to `properties(v).tag.name`, `nodes(p)[0].tag.name`, etc.

Reply:
If I understood what you mean correctly, all tck cases calling the 'properties' function runs as expected for the time being. I didn't find any abnormal case, at least. Let me know if any tck case needs to be updated.

If we want to force all properties returned by this function to report tag information, we probabaly need to check with PD and open a new issue.

Reply:
For these tck cases, I added tags into them, otherwise errors would be reported.

Reply:
Essentially, it's all the same issue, that is, the tag must be specified to get the attribute. 
"all tck cases calling the 'properties' function runs as expected for the time being." just because the test case is not covered the exception scenario.